packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_RelationalConnectionBuilderHelper.ts

```diff
@@ -213,6 +215,28 @@ export const V1_buildDatasourceSpecification = (
       protocol.region,
     );
     return redshiftSpec;
+  } else if (protocol instanceof V1_SpannerDatasourceSpecification) {
+    assertNonNullable(
```

Comment:
Let's use `assetNonEmptyString` for these fields validation and the error message should then be `... is missing or empty`